packages/contentful--app-scripts/src/create-app-definition/create-app-definition.ts

```diff
@@ -55,7 +56,7 @@ function assertValidArguments(accessToken, appDefinitionSettings) {
   }
 }
 
-async function createAppDefinition(accessToken, appDefinitionSettings = { locations: [] }) {
+export async function createAppDefinition(accessToken: string, appDefinitionSettings: AppDefinitionSettings) {
```

Comment:
We previously had a default value for `appDefinitionSettings`. Now we only have a default value for `appDefinitionSettings.fields`. Could this cause issues for some code paths or is `appDefinitionSettings` always defined`?

Reply:
Yeah, we ask for them in interactive mode, so they are always present.